src/Microsoft.Diagnostics.ExtensionCommands/EEHeap.cs

```diff
@@ -0,0 +1,506 @@
+﻿using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "eeheap", Aliases = new string[] { "EEHeap" }, Help = "Displays information about native memory that CLR has allocated.")]
+    public class EEHeap : CommandBase
+    {
+        [ServiceImport]
+        public IRuntimeService RuntimeService { get; set; }
+
+        [ServiceImport]
+        public IMemoryService MemoryService { get; set; }
+
+        [Option(Name = "--gc", Aliases = new string[] { "-gc" }, Help = "Only display the GC.")]
+        public bool ShowGC { get; set; }
+
+        [Option(Name = "--loader", Aliases = new string[] { "-loader" }, Help = "Only display the Loader.")]
+        public bool ShowLoader { get; set; }
+
+        public override void Invoke()
+        {
+            IRuntime[] runtimes = RuntimeService.EnumerateRuntimes().ToArray();
+
+            ulong totalBytes = 0;
+            StringBuilder stringBuilder = null;
+            foreach (IRuntime iRuntime in RuntimeService.EnumerateRuntimes())
```

Comment:
You can use "runtimes" here instead of calling EnumerateRuntimes() again (unless I'm missing something).

Reply:
Just an artifact of bad refactoring, fixed.